### PR TITLE
Add server addresses page

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Add a page with server addresses for the different deployments.

#### Screenshots

Bildschirmfoto 2021-07-05 um 19 03 08

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Review by adriansmares — APPROVED** [review]

**Review by kschiffer — APPROVED** [review]

**Review by neoaggelos — APPROVED** [review]

File: doc/content/getting-started/server-addresses/_index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+**Community Edition**: See [Community Edition Addresses]({{< ref "getting-started/ttn/addresses" >}})
+
+**Enterprise and Open Source**: This is the domain you configure when installing {{% tts %}}. If you followed the [Getting Started guide]({{< ref "/getting-started" >}}), this is what you use instead of `thethings.example.com`.
+
+## Console
+
+To access the Console, simply enter the server address of your deployment in a browser.
+
+For example, https://eu1.cloud.thethings.network for the `eu1` Community Edition Console, or https://tenant.au1.thethings.industries for the `au1` Cloud Console, where `tenant` is your Tenant ID. If you installed {{% tts %}} Enterprise or Open Source, access the Console at the domain you used in place of https://thethings.example.com.
```
Comment (johanstokking) [inline doc/content/getting-started/server-addresses/_index.md:27]: Urls must be in backticks

File: doc/content/getting-started/server-addresses/_index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## LNS
+
+The LNS gateway address is a combination of the **protocol** (wss), the **server address**, and the **port** (8887).
+
+For example, the `eu1` Community Edition LNS address is wss://eu1.cloud.thethings.network:8887.
+
+The LNS address for an `au1` Cloud tenant is wss://tenant.au1.thethings.industries:8887, where `tenant` should be replaced with your Tenant ID.
```
Comment (johanstokking) [inline doc/content/getting-started/server-addresses/_index.md:35]: This is also an example. Best is to put this in a highlight box that makes it really clear that these are examples.

File: doc/content/getting-started/server-addresses/_index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+If you installed {{% tts %}} Enterprise or Open Source, the LNS address is the domain you used in configuration, for example wss://thethings.example.com:8887.
+
+## CUPS
+
+The CUPS gateway address is a combination of the **protocol** (https), the **server address**, and the **port** (443).
+
+For example, the `eu1` Community Edition CUPS address is https://cloud.eu1.thethings.network:443.
+
+The CUPS address for an `au1` Cloud tenant is https://tenant.au1.thethings.industries:443, where `tenant` should be replaced with your Tenant ID.
```
Comment (johanstokking) [inline doc/content/getting-started/server-addresses/_index.md:45]: Idem

File: doc/content/getting-started/installation/_index.md [hunk 4]
```diff
@@ -2,10 +2,11 @@
 title: "Installing The Things Stack"
 description: ""
 distributions: ["Enterprise", "Open Source"]
-weight: -1
 ---
 
-This is a guide for setting up a private LoRaWAN network server using {{% tts %}} for LoRaWAN.
+This is a guide for installing {{% tts %}} Enterprise or Open Source on your own hardware, to run your own private LoRaWAN network server.
+
+To use {{% tts %}} in the cloud without installing it on your own hardware, see [{{% tts %}} Cloud]({{< relref "cloud-hosted" >}}) (our professional, SLA backed offering) or [The Things Network]({{< relref "ttn" >}}) (our free-to-use community edition).
```
Comment (nejraselimovic) [inline doc/content/getting-started/installation/_index.md:9]:
```suggestion
To use {{% tts %}} in the cloud without installing it on your own hardware, see [{{% tts %}} Cloud]({{< relref "cloud-hosted" >}}) (our professional, SLA backed offering) or [{{% tts %}} Community Edition]({{< relref "community-edition" >}}) (our free-to-use community edition).
```

File: doc/content/getting-started/upgrading/_index.md [hunk 5]
```diff
@@ -2,10 +2,11 @@
 title: "Upgrading The Things Stack"
 description: ""
 distributions: ["Enterprise", "Open Source"]
-weight: 3
 ---
 
-Follow these instructions to upgrade an existing {{% tts %}} instance.
+This section contains instructions to upgrade a {{% tts %}} Enterprise or Open Source deployment, that was installed using the [installation guide]({{< relref "installation" >}}).
+
+{{% tts %}} Cloud and Community Edition are updated automatically (you don't need to do anything!)
```
Comment (nejraselimovic) [inline doc/content/getting-started/upgrading/_index.md:9]:
```suggestion
{{% tts %}} Cloud and Community Edition are updated automatically (you do not need to do anything!)
```

**Review by nejraselimovic — APPROVED** [review]

File: doc/content/getting-started/server-addresses/_index.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+title: "Server Addresses"
+description: ""
+---
+
+To access the [Console]({{< ref "getting-started/console" >}}), register a [command line client]({{< ref "getting-started/cli" >}}), [connect gateways]({{< ref "gateways/adding-gateways" >}}), or schedule uplinks via [webhooks]({{< ref "integrations/webhooks" >}}), it is necessary to know the server address of your {{% tts %}} instance. This section contains information about addresses for different {{% tts %}} deployments.
+
+<!--more-->
+
+By default, all components of {{% tts %}}, such as the Gateway Server, Identity Server, and Console, are located at the same server address.
+It is possible to configure separate addresses for the different components if you are running Enterprise or Open Source. For Cloud and Community Edition, all endpoints are located at the same server address (except the Identity Server, which is [only available]({{< ref "getting-started/cloud-hosted/addresses#api-endpoints" >}}) in the `eu1` cluster). The server address for the different {{% tts %}} deployments are listed below.
```
Comment (johanstokking) [inline doc/content/getting-started/server-addresses/_index.md:11]: This first sentence isn't really true and is a bit confusing with what comes next (which is true).

Maybe rephrase to:

```suggestion
Deployments of {{% tts %}} in a single cluster use the same server address for all components.
Distributed deployments, like {{% tts %}} Cloud and Community Edition, use different addresses per region for routing components, while the address of global components (like Identity Server) is always the same. Enterprise and Open Source deployments may be in a single cluster, or distributed over multiple clusters with multiple addresses.
The server address for the different {{% tts %}} deployments are listed below.
```

File: doc/content/getting-started/what-is-tts/_index.md [hunk 7]
```diff
@@ -12,7 +12,15 @@ Welcome to {{% tts %}}! In this section, we help you to understand how to use {{
 
 {{% tts %}} is an enterprise grade LoRaWAN network server, built on an [open source](https://github.com/TheThingsNetwork/lorawan-stack) core. {{% tts %}} allows you to build and manage LoRaWAN networks on your own hardware or in the cloud.
 
-To get started running {{% tts %}} on your own, see the [installation section]({{< ref "/getting-started/installation" >}}). To inquire about a worry-free hosted solution, [get in touch with us](mailto:info@thethingsindustries.com).
+### {{% tts %}} Deployments
+
+{{% tts %}} comes in many different flavors to meet different requirements. For all available deployments, see [Get {{% tts %}}]({{< ref "download" >}}). The most common deployments are:
```
Comment (johanstokking) [inline doc/content/getting-started/what-is-tts/_index.md:17]: People can't download Cloud. That is Open Source only which is not listed here. Maybe add a fourth option `**Open Source**` with the download link?

Comment (benolayinka) [reply doc/content/getting-started/what-is-tts/_index.md:17]: Download describes all the deployments, the title is a relic.

Bildschirmfoto 2021-07-08 um 14 33 39
